app/styles/components/filter-sidebar-label.scss

```
@@ -0,0 +1,11 @@
+.filter-sidebar-label {
+  display: block;
+  font-size: 0.75rem;
+  font-weight: 500;
+  padding: 0.5rem 0;
+}
+
+> * {
+  font-weight: 300;
+  margin-top: 0.3rem;
+}
```

This is will add the styling to **all child elements of the current selector**.
```suggestion
.filter-sidebar-label > * {
  font-weight: 300;
  margin-top: 0.3rem;
}
```

**or **
indent in parent block.